fili-sql/src/main/java/com/yahoo/bard/webservice/sql/presto/PrestoSqlBackedClient.java

```diff
@@ -138,9 +148,11 @@ private JsonNode executeAndProcessQuery(DruidAggregationQuery<?> druidQuery) {
      * </ul>
      *
      * @param sqlQuery The sql query converted from druid query.
+     * @param timestampFormat The format used to parse the timestamp
      * @return a presto compatible sql dialect.
      */
-    protected static String sqlQueryToPrestoQuery(String sqlQuery) {
+    protected static String sqlQueryToPrestoQuery(String sqlQuery, String timestampFormat) {
+        LOG.info("using fili version of PrestoSqlBackedClient");
```

Comment:
Maybe delete this? Or maybe change to not use `LOG.info`

Reply:
good catch. removed the logging